opennms-webapp-rest/src/main/java/org/opennms/web/rest/v2/EventConfPersistenceService.java

```diff
@@ -108,6 +111,37 @@ public void enableDisableConfSourcesEvents(final Long sourceId, final EnableDisa
         eventConfEventDao.updateEventEnabledFlag(sourceId,enableDisableConfSourceEventsPayload.getEventsIds(),enableDisableConfSourceEventsPayload.isEnable());
     }
 
+
+    @Transactional
+    public void updateEventConfEvent(final Long eventId, EventConfEventPayload payload) {
```

Comment:
Not sure if I understand this correctly, EventConfEventPayload shouldn't be limited to two or three fields. It should be complete event conf in xml format.

User can update whole event.